shared/components/line-chart/line-chart.component.ts

```diff
@@ -12,39 +30,80 @@ import { observeResize } from '@shared/utils/observe-resize';
   styleUrls: ['./line-chart.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class LineChartComponent implements AfterViewInit {
+export class LineChartComponent implements AfterViewInit, OnChanges {
+  @ContentChild(LineChartTooltipDirective, { static: false, read: TemplateRef }) tooltipRef: TemplateRef<{}>;
+
+  @ViewChild('tooltipContainer', { static: true }) tooltipContainerRef: ElementRef;
   @ViewChild('chart', { static: false }) chartRef: ElementRef;
 
   @Input() public data: ChartPoint[];
-  @Input() private color: string;
+  @Input() private color: string = '#000000';
```

Comment:
May be u can use CSS `currentColor` power here and set property color in the parent component